### ISSUE TITLE
How to give description while creating new issue

### ISSUE THREAD

**Issue by deepthijohn** [toabctl/jiracli#51]
where to specify description
./jiracli  -c PROJECT "Dev Bug" "my test summary" "abc,def" "xxx,yyyy"

**Comment by toabctl** [comment]
@deepthijohn a text editor will popup and ask for the description. Do you need to set the description via the command call?

**Comment by deepthijohn** [comment]
yes. i want to add description through command call. and it would be better if i can add description while creating issue.
